### DISCUSSION TITLE
I haven't gotten blocks for a long time.

### DISCUSSION THREAD

**Discussion by paar13kr** [filecoin-project/lotus#8489] [Storage Provider] [answered]
Miner id : f01107182
I got 5 to 7 blocks every month, but I haven't gotten them for too long.
There was no log that missed the block, and it seems that the wining post is still working properly.
Why can't I get a block?

2022-04-14T18:45:36.121+0900	INFO	miner	miner/miner.go:479	completed mineOne	{"tookMilliseconds": 11, "forRound": 1720772, "baseEpoch": 1720771, "baseDeltaSeconds": 6, "nullRounds": 0, "lateStart": false, "beaconEpoch": 1816616, "lookbackEpochs": 900, "networkPowerAtLookback": "18597239650194915328", "minerPowerAtLookback": "252887674388480", "isEligible": true, "isWinner": false, "error": null}

**Comment by wyfor4667** [comment]
I haven't gotten blocks too! Even I have a little faults.
But it's abnormal

**Comment by rjan90** [comment, marked as answer]
Hey!

As long as you have not passed the `projected block win with 99.9% threshold` time, there should not be any problems if you still are in sync / not missed windowPoSt / not seen any lost blocks,  and you see `"isEligible": true` in your logs.

For example:
```
Projected average block win rate: 3.40/week (every 49h26m34s)
Projected block win with 99.9% probability every 341h30m37s
```
In this example the Storage Provider should on _avarage_ get 3.40 blocks per week, but the upper boundary might be 341 hours (14 days) to get one.

**Reply by haheihou5** [reply]
我的节点也很长时间不爆块了,并且已经超过了最大上限时间好几天了

**Reply by haheihou5** [reply]
miner节点是f01390330

**Reply by haheihou5** [reply]
我的问题已经解决了,是磁盘io的问题,导致出块异常,检查一下你的磁盘io,不要有其他程序或行为占用了磁盘io
